src/player.cpp

```diff
@@ -5116,6 +5095,139 @@ int player::hitall(int dam, int vary, Creature *source)
     return damage_taken;
 }
 
+float player::fall_damage_mod() const
+{
+    float ret = 1.0f;
+
+    // Ability to land properly is 2x as important as dexterity itself
+    float dex_dodge = dex_cur / 2 + get_skill_level( "dodge" );
+    // Penalize for wearing heavy stuff
+    dex_dodge -= ( ( encumb(bp_leg_l) + encumb(bp_leg_r) ) / 20 ) + ( encumb(bp_torso) / 10 );
+    // But prevent it from increasing damage
+    dex_dodge = std::min( 0.0f, dex_dodge );
```

Comment:
That means `dex_doged` will be negative or 0. Without any encumbrance, this will be 0 (skill and stats are always positive). It will do nothing. Should this be `std::max`?